docker/build.mk

```diff
@@ -216,6 +216,7 @@ define fuzzer_oss_fuzz_benchmark_template
     --file=docker/oss-fuzz-builder/Dockerfile \
     --build-arg parent_image=$(BASE_TAG)/oss-fuzz/builders/$(1)/$($(2)-project-name)-intermediate \
     --build-arg fuzzer=$(1) \
+    --build-arg benchmark=$(2) \
```

Comment:
This probably needs to be done in the config files in `experiment/gcs` as well

Reply:
Yes, this is in my CL :)

Reply:
Ah didn't see.